lib/node_modules/@stdlib/blas/ext/base/dnanasumors/test/test.ndarray.js

```diff
@@ -77,6 +77,18 @@ tape( 'the function calculates the sum of absolute values (ignoring NaN values)'
 	t.end();
 });
 
+tape( 'if provided a `stride` and `offset` parameter equal to `0` and the first element is NaN, the function returns `0`', function test( t ) {
```

Comment:
@Neerajpathak07 I believe this test needs to also be replicated across the other test files. While the test may get us to 100% coverage, for completeness, we should test that the other execution contexts also return expected results with the `stride` is equal to `0`.